django/conf/global_settings.py

```diff
@@ -502,6 +502,17 @@
 CACHE_MIDDLEWARE_SECONDS = 600
 CACHE_MIDDLEWARE_ALIAS = 'default'
 
+#########
+# TASKS #
+#########
+
+# The task backends to use.
+QUEUES = {
```

Comment:
It would probably be an internal configuration of the backend. Or the backend would have to be written with global connection pooling in mind.